supervisor/dbus/payloads/interface_update.tmpl

```diff
@@ -61,7 +61,7 @@
     ,
     '802-3-ethernet':
         {
-            'assigned-mac-address': <'stable'>
+            'assigned-mac-address': <'preserve'>
```

Comment:
Is there a reason why we set this in first place? Can we not set it at all and leave it up to NM?

Reply:
Different NM working differently based on his version. The platform should work everywhere exactly in the same way and eliminate customizing. Also if we update NM on OS, it should not change his behavior after the update.